bigquery/api/GettingStarted/test/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("test")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]

```

Comment:
Add license header and populate AssemblyCompany and AssemblyCopyright ([here's what we're using in google-api-dotnet-client](https://github.com/google/google-api-dotnet-client/blob/master/Src/GoogleApis.Core/Properties/AssemblyInfo.cs#L26-L28)).